go.sum

```diff
@@ -1,4 +1,8 @@
 github.com/FortAwesome/Font-Awesome v0.0.0-20230327165841-0698449d50f2/go.mod h1:IUgezN/MFpCDIlFezw3L8j83oeiIuYoj28Miwr/KUYo=
+github.com/FortAwesome/Font-Awesome v0.0.0-20240402185447-c0f460dca7f7/go.mod h1:IUgezN/MFpCDIlFezw3L8j83oeiIuYoj28Miwr/KUYo=
```

Comment:
Why does this have checksums for multiple versions? If we updated a dependency, we do not need to keep the old checksums for versions we no longer depend on. Is this automatically generated? Maybe delete it and regenerate it?